docs/libcudacxx/ptx/instructions/generated/shfl_sync.rst

```diff
@@ -70,8 +70,11 @@ shfl.sync
 
 **Constrains and checks**
 
+The following conditions are checked at *run-time* in debug mode:
+
 - ``T`` must have 32-bit size (compile-time)
-- ``lane_idx_offset`` must be less than the warp size (debug mode)
-- ``clamp_segmask`` must use the bit positions [0:4] and [8:12] (debug mode)
-- ``lane_mask`` must be a subset of the active mask (debug mode)
-- The destination lane must be a member of the ``lane_mask`` (debug mode)
+- ``lane_idx_offset`` must be less than the warp size
+- ``clamp_segmask`` must use the bit positions [0:4] and [8:12])
+- ``lane_mask`` must be a subset of the active mask
+- All lanes must have the same value for ``lane_mask``
+- The destination lane must be a member of the ``lane_mask``
```

Comment:
These are changes to a generated file. Those will be overwritten and lost the next time we generate PTX instructions. Please move hand-written documentation elsewhere.